systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -1062,8 +1062,10 @@
       end
       
       model = applyToAllRigidBodyElements(model,'updateParams',fr.getPoly,p);
-
-      model = compile(model);
+      
+      if isnumeric(p)
+        model = compile(model);
+      end
```

Comment:
i worry that we might be missing important steps here.  what failed exactly in the compile for non-numeric?  (maybe we can leave something out more selectively)?